packages/css/src/source-maps.tsx

```diff
@@ -15,21 +9,21 @@ const getFileName = (sourceFile: ts.SourceFile): string => {
  *
  * Will return something like `/*# sourceMappingURL=...`
  */
-export function getSourceMap(
+export function buildSourceMap(
   offset: {
     line: number;
     character: number;
   },
-  sourceFile: ts.SourceFile,
-  context: ts.TransformationContext
+  code: string,
```

Comment:
this file not currently used but i make it not need TS anymore for when we implement this again.